static/app/views/onboarding/otherSetup.tsx

```diff
@@ -0,0 +1,160 @@
+import 'prism-sentry/index.css';
+
+import * as React from 'react';
+import styled from '@emotion/styled';
+import {motion} from 'framer-motion';
+
+import {openInviteMembersModal} from 'app/actionCreators/modal';
+import {Client} from 'app/api';
+import Alert from 'app/components/alert';
+import AsyncComponent from 'app/components/asyncComponent';
+import Button from 'app/components/button';
+import ExternalLink from 'app/components/links/externalLink';
+import {PlatformKey} from 'app/data/platformCategories';
+import {t, tct} from 'app/locale';
+import {Organization, Project} from 'app/types';
+import {analytics} from 'app/utils/analytics';
+import getDynamicText from 'app/utils/getDynamicText';
+import withApi from 'app/utils/withApi';
+import withOrganization from 'app/utils/withOrganization';
+import {ProjectKey} from 'app/views/settings/project/projectKeys/types';
+
+import FirstEventFooter from './components/firstEventFooter';
+import SetupIntroduction from './components/setupIntroduction';
+import {StepProps} from './types';
+
+type AnalyticsOpts = {
+  organization: Organization;
+  project: Project | null;
+  platform: PlatformKey | null;
+};
+
+const recordAnalyticsDocsClicked = ({organization, project, platform}: AnalyticsOpts) =>
+  analytics('onboarding_v2.full_docs_clicked', {
+    org_id: organization.id,
+    project: project?.slug,
+    platform,
+  });
+
+type Props = StepProps & {
+  api: Client;
+  organization: Organization;
+} & AsyncComponent['props'];
+
+type State = {
+  keyList: Array<ProjectKey> | null;
+} & AsyncComponent['state'];
+
+class OtherSetup extends AsyncComponent<Props, State> {
+  getDefaultState() {
+    return {
+      ...super.getDefaultState(),
+      keyList: null,
+    };
+  }
+
+  getEndpoints(): ReturnType<AsyncComponent['getEndpoints']> {
+    const {organization, project} = this.props;
+    return [['keyList', `/projects/${organization.slug}/${project?.slug}/keys/`]];
+  }
+
+  handleFullDocsClick = () => {
+    const {organization, project, platform} = this.props;
+    recordAnalyticsDocsClicked({organization, project, platform});
+  };
+
+  render() {
+    const {organization, project} = this.props;
+    const {keyList} = this.state;
+
+    const currentPlatform = 'other';
+
+    const introduction = (
```

Comment:
@roggenkemper Since we have the same component in `documentationSetup`, might be worth converting this into a component that we can reuse there as well. You can add the component here: `static/app/views/onboarding/components`

Reply:
lots of repeated code there. I made the introduction part a new component and am using that in both other and document setup